dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -1421,6 +1441,16 @@ std::unordered_map<TableID, DAGStorageInterpreter::StorageWithStructureLock> DAG
             }
         }
 
+        if (!check_storages_schema_same(table_storages))
+        {
+            // Since we can not know which table's schema is newer, we need to sync all tables' schema.
+            need_sync_table_ids.clear();
+            need_sync_table_ids.append_range(table_scan.getPhysicalTableIDs());
```

Comment:
[📡 STDERR] /Users/pingcap/workspace/bp-tiflash-release-darwin-amd64-l7qjb-build-binaries/source/tiflash/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp:1448:33: error: no member named 'append_range' in 'std::vector<long long>'
    [📡 STDERR]             need_sync_table_ids.append_range(table_scan.getPhysicalTableIDs());
    [📡 STDERR]             ~~~~~~~~~~~~~~~~~~~ ^
    [📡 STDERR] 1 error generated.